File: app/src/main/java/com/ignis/android_cleanarchitecture/domain/model/LinkModel.java
package com.ignis.android_cleanarchitecture.domain.model;

import io.realm.RealmObject;

/**
 * Link Model
 * <p>
 * Created by tamura_k on 2016/06/03.
 */
public class LinkModel extends RealmObject {

    //市区町村名
    private String name;
    //対応するlivedoor 天気情報のURL
    private String link;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getLink() {
        return link;
    }

    public void setLink(String link) {
        this.link = link;
    }

}

File: app/src/main/java/com/kazakago/cleanarchitecture/domain/model/weather/LinkModel.kt
package com.kazakago.cleanarchitecture.domain.model.weather

import paperparcel.PaperParcel
import paperparcel.PaperParcelable

@PaperParcel
data class LinkModel(
        //市区町村名
        val name: String,
        //対応するlivedoor 天気情報のURL
        val link: String
) : PaperParcelable {

    companion object {
        @JvmField
        val CREATOR = PaperParcelLinkModel.CREATOR
    }

}
